### ISSUE TITLE
Enable non-blob async writeback

### ISSUE THREAD

**Issue by rclapp** [cubewise-code/tm1py#921]
One of the trade offs of using a blob based write back is that you can no longer leverage change sets. In certain applications change sets are required to properly group transactions together. We should expose useBlob parameters in the async write back methods so that change sets could be used. If we do this we may need to change the way change sets are managed. An async writer could

1. Return a list of change sets, 1 from each worker
2. Create a change set and pass it into the worker, then all workers write using the same change set ID

Thoughts?

**Comment by MariusWirtz** [comment]
I agree. Using changesets in conjunction with the fast async write would be nice.
Ideally, we could make use of changesets in TI.  If that's not possible we could use the standard REST-based write. 

In both cases, we could return a list of changesets or share a changeset-id among the workers.
I will tag this issue as an enhancement.
